test/hotspot/jtreg/serviceability/sa/sadebugd/ClhsdbAttachToDebugServerWithCommandLine.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021 NTT DATA.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.PrintStream;
+
+import jdk.test.lib.JDKToolLauncher;
+import jdk.test.lib.apps.LingeredApp;
+import jdk.test.lib.process.OutputAnalyzer;
+import jdk.test.lib.SA.SATestUtils;
+
+import jtreg.SkippedException;
+
+/**
+ * @test
+ * @bug 8263342
+ * @requires vm.hasSA
+ * @requires os.family != "windows"
+ * @library /test/lib
+ * @run main/othervm ClhsdbAttachToDebugServerWithCommandLine
+ */
+
+public class ClhsdbAttachToDebugServerWithCommandLine {
```

Comment:
The test name is rather long. Since this test is already in the `sadebugd` directory, you could shorten it to `ClhsdbAttachWithCommandLine`, although I'm not so sure I like it either. How about `ClhsdbAttachWithConnect` or `ClhsdbAttachWithConnectArgument`, or `ClhsdbTestConnectArgument` (I think I like the last one best)?